common/scripted_effects/wc_unit_scripted_effects.txt

```diff
@@ -151,4 +151,216 @@ spawn_death_knight_troops_effect = {
 		maintenance = no
 		earmark = death_knights_forces
 	}
+}
+spawn_cthun_guard_troops_effect = {
```

Comment:
It's a bigger version of `spawn_cthun_follower_troops_effect`. Why not to use a few `spawn_cthun_follower_troops_effect` inside?